kolibri/core/assets/src/kolibri_module.js

```diff
@@ -1,32 +1,33 @@
-'use strict';
 /**
  * KolibriModule module.
  * Provides the base class for Kolibri Modules - all Kolibri Modules must extend this Base class.
  * @module kolibri_module
  */
 
-var Kolibri = require('kolibri');
-var _ = require('lodash');
-var Backbone = require('backbone');
+const Kolibri = require('./core_app');
+const _ = require('lodash');
+const Backbone = require('backbone');
 
 /**
  * The constructor function for the base KolibriModule object.
  * @param {object} options - an options hash to set properties of the object.
  * @constructor
  */
-var KolibriModule = function (options) {
```

Comment:
This can actually be changed to an ES6 class definition too, if you are feeling like it.